docs/integration/integration-more.md

```diff
@@ -0,0 +1,163 @@
+
+## Official wallets
+
+Official desktop wallet: https://github.com/alephium/desktop-wallet/releases/latest
+
+Official mobile wallet:
+* Android: https://play.google.com/store/apps/details?id=org.alephium.wallet
+* Apple: https://apps.apple.com/us/app/alephium-wallet/id6469043072
+
+Official web extension wallet:
+* Chrome: https://chrome.google.com/webstore/detail/alephium-extension-wallet/gdokollfhmnbfckbobkdbakhilldkhcj
+* Firefox: https://addons.mozilla.org/en-US/firefox/addon/alephiumextensionwallet/
+
+## Fullnode 
+
+### Node installation documents
+
+[Getting started with Fullnode](../full-node/getting-started.md)
+
+### Required disk size
+
+The current mainnet database size is around 85G. Documentation for [pruned nodes](../full-node/full-node-more#pruning).
+
+### Mainnet snapshots
+
+URL: [https://archives.alephium.org/#mainnet/](https://archives.alephium.org/#mainnet/)
+
+To use the snapshot, use the following [documentation](../full-node/loading-snapshot.md).
+
+### Block producing rate
+
+Currently the average network block production rate is 1 block every 4sec.
+This is because the network currently has 16 chains each with a blocktime of 64sec. (64/16=4s in average).
+
+The next network upgrade will reduce blocktime to 16s for all 16 chains resulting in the average network block production rate to be 1s.
+
+### Configure file directory
+
+Data folder for fullnode can be changed, use the following [documentation](../full-node/full-node-more.md#moving-the-alephium-data-folder).
+
+### How to customize RPC Port and directory of block data
+
+For customize the directory of block data, you can refer to the doc [here](../full-node/full-node-more.md#moving-the-alephium-data-folder).
+
+For customize the RPC port: add this line `alephium.network.rest-port = PORT` to the `user.conf` file.
+
+### RPC/SDK documentation link
+
+After running the full node, you can access the API doc at `http://full-node-ip:port/docs`.
+
+[JS/TS SDK documentation](https://github.com/alephium/alephium-web3).
+
+### Does the wallet node need to expose its port to the outside? If so, is it acceptable for the machine to have a NAT IP?
+
+Yes and NAT IP is ok.
+
+### Is it necessary to add our node IP into your whitelist when synchronizing?
+
+No, it’s not necessary.
+
+
+## Wallets & Addresses
+
+### Validate address format
+
+Address is base58 encoded string, there is no max length for address, but in most case the decoded address length is 33 bytes.
+
+There is a utility function in the TS SDK for it: `https://github.com/alephium/alephium-web3/blob/v0.23.0/packages/web3/src/utils/exchange.ts#L23-L41`.
+
+### Access the address history
+
+You can access the address history in two ways:
+* You can get the address history from the explorer, but you will need to run your own explorer-backend, installation [documentation](../explorer-backend/getting-started). Endpoint: `https://backend-v113.mainnet.alephium.org/docs/#/Addresses/getAddressesAddressTransactions`.
```

Comment:
We can use this link:
https://backend.mainnet.alephium.org/docs/#/Addresses/getAddressesAddressTransactions